torchdata/dataloader2/reading_service.py

```diff
@@ -102,24 +101,42 @@ def _collate_no_op(batch):
 class _IterateQueueDataPipes:
```

Comment:
Why don't we make it as a `IterDataPipe` then we don't need to call `IterableWrapper` [here](https://github.com/pytorch/data/blob/86df1a09c0f649aca195a233508669de35b8623b/torchdata/dataloader2/reading_service.py#L166)?

Reply:
Historical reasons changed now.